multimedia/gphoto2

```diff
@@ -0,0 +1 @@
+Subproject commit 0b531bb70622a9d88b2fc0cfb0c649d0b6400f76
```

Comment:
Looks like a subproject module. I've queried it in a thread here:
[https://forum.openwrt.org/viewtopic.php?id=56272](https://forum.openwrt.org/viewtopic.php?id=56272)